pkg/controllers/tagging/tagging_controller.go

```diff
@@ -32,9 +32,14 @@ import (
 	opt "k8s.io/cloud-provider-aws/pkg/controllers/options"
 	awsv1 "k8s.io/cloud-provider-aws/pkg/providers/v1"
 	nodehelpers "k8s.io/cloud-provider/node/helpers"
+	_ "k8s.io/component-base/metrics/prometheus/workqueue" // enable prometheus provider for workqueue metrics
```

Comment:
oh cool, didn't realize this was all done with side effects via `init`

Reply:
Yeah assuming these built-in metrics do the job, I'd like to get rid of the other ones we have after 1.29 is cut. The `workqueue` should track processing time (dequeue latency), errors, size, etc.